build/Common.props

```diff
@@ -27,13 +27,11 @@
 
   <ItemGroup>
     <PackageReference Include="Microsoft.NETFramework.ReferenceAssemblies" PrivateAssets="All" />
-    <PackageReference Include="StyleCop.Analyzers" Condition="'$(SkipAnalysis)'!='true'">
-      <PrivateAssets>All</PrivateAssets>
-    </PackageReference>
+    <PackageReference Include="StyleCop.Analyzers" PrivateAssets="All" Condition="'$(SkipAnalysis)'!='true'" />
     <PackageReference Include="Microsoft.CodeCoverage" PrivateAssets="All" Condition="'$(Configuration)'=='Release'" />
-    <!--<PackageReference Include="Microsoft.CodeAnalysis.FxCopAnalyzers" Condition="'$(SkipAnalysis)'!='true'">
-      <PrivateAssets>all</PrivateAssets>
-    </PackageReference>-->
+    <!--
+    <PackageReference Include="Microsoft.CodeAnalysis.FxCopAnalyzers" PrivateAssets="All" Condition="'$(SkipAnalysis)'!='true'" />
```

Comment:
This should be an editorial change, note that previously we were using "All" and "all", inline vs. property without consistency (which makes it hard to search).